asset_dashboard/static/js/components/PortfolioTable.js

```diff
@@ -1,20 +1,17 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import ReactTable from './BaseTable'
 import Button from 'react-bootstrap/Button'
 import Form from 'react-bootstrap/Form'
 import InputGroup from 'react-bootstrap/InputGroup'
 import projectColumns from './table_utils/projectColumns'
+import SubRow from './table_utils/SubRow'
 
 const PortfolioTable = ({ portfolio, onRemoveFromPortfolio, savePortfolioName, savePortfolio, createNewPortfolio }) => {
-  const [edit, setEdit] = useState(portfolio.name ? false : true)
-
-  const onRowClick = ({ original }) => {
-    return {
-      onClick: e => {
-        onRemoveFromPortfolio(original)
-      }
-    }
-  }
+  const [edit, setEdit] = useState(false)
+  
+  useEffect(() => {
+    setEdit(portfolio.id == null ? true : false)
+  }, [portfolio])
```

Comment:
fixed this so the text input wasn't active when the page loads with an existing portfolio.